### PR TITLE
Add Chao and Phung to empiricals

### PR DESCRIPTION
**Pull request by jpa198**
I'm guessing this is how the numbering is supposed to work for OQ empiricals?

### CODE REVIEW

File: empirical/util/classdef.py [hunk 1]
```diff
@@ -90,6 +90,8 @@ class GMM(ExtendedEnum):
     G_17 = 1031
     BC_16 = 1041
     S_16 = 1051
+    PH_20 = 1061
```
Comment (jasonmotha) [inline empirical/util/classdef.py:93]: Because these ones are ASC/SI/SS it probably is worth renaming it to PH_20_CR

**Review by jasonmotha — CHANGES_REQUESTED** [review]

File: empirical/util/classdef.py (hunk 1 above)
Comment (jpa198) [reply empirical/util/classdef.py:93]: Do you think it's worth adding the SI/SS now?

Comment (jasonmotha) [reply empirical/util/classdef.py:93]: Well, yeah. But not a big issue.

**Comment by jasonmotha** [comment]
Tests failing because pickle files not found.

**Comment by jpa198** [comment]
> Tests failing because pickle files not found.

The files are there. Submitted an empty commit to trigger a re run. Odd though.

**Comment by jasonmotha** [comment]
> > Tests failing because pickle files not found.
> 
> The files are there. Submitted an empty commit to trigger a re run. Odd though.

lol. Still not there apparently /var/lib/jenkins/workspace/Empirical_Engine/empirical/test/sample0/output/as_2016_ret_val_rrup_10_mag_6.25_Ds595.P

**Review by jasonmotha — APPROVED** [review]

**Review by jasonmotha — CHANGES_REQUESTED** [review]
Whoops still waiting on the rename.

**Comment by jpa198** [comment]
> Whoops still waiting on the rename.

Oops, thought I did that yesterday

File: empirical/util/openquake_wrapper.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
     OQ = False
 
 # GMM numbers to match empirical.util.classdef.GMM
-OQ_GMM = [1012, 1013, 1021, 1022, 1023, 1031, 1041, 1051]
+OQ_GMM = [1012, 1013, 1021, 1022, 1023, 1031, 1041, 1051, 1061, 1062]
```
Comment (viktor76525) [inline empirical/util/openquake_wrapper.py:21]: 3 number is model, last one is submodel asc

**Review by viktor76525 — APPROVED** [review]

**Review by jasonmotha — APPROVED** [review]

**Pull request merged**